libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx

```diff
@@ -77,7 +78,8 @@ export const CompilerContainer = (props: CompilerContainerProps) => {
           includeNightlies: api.getConfiguration('includeNightlies') || false,
           optimise: (optimize !== null) && (optimize !== undefined) ? optimize : api.getConfiguration('optimise') || false,
           runs: (runs !== null) && (runs !== 'null') && (runs !== undefined) && (runs !== 'undefined') ? runs : 200,
-          evmVersion: (evmVersion !== null) && (evmVersion !== 'null') && (evmVersion !== undefined) && (evmVersion !== 'undefined') ? evmVersion : 'default'
+          evmVersion: (evmVersion !== null) && (evmVersion !== 'null') && (evmVersion !== undefined) && (evmVersion !== 'undefined') ? evmVersion : 'default',
+          language: (language !== null) && (language !== 'null') && (language !== undefined) && (language !== 'undefined') ? language : 'Solidity'
```

Comment:
what if it is not 'Yul'?

Reply:
So we should ensure that it's either `Yul` or `Solidity` ?